Tensile/TensileCreateLibrary.py

```diff
@@ -207,7 +207,8 @@ def buildSourceCodeObjectFile(CxxCompiler, outputPath, kernelFile):
       # for hipclang 5.2 and above, clang offload bundler changes the way input/output files are specified
       inflag = "-inputs"
       outflag = "-outputs"
-      if (hipccMaj == 5 and hipccMin >= 2) or hipccMaj >= 6:
+      # clang 15.0.0 still wants inputs on windows so the hipcc version condition doesn't work
+      if os.name != "nt" and ((hipccMaj == 5 and hipccMin >= 2) or hipccMaj >= 6):
```

Comment:
This is the sort of thing that would normally be checked at configuration-time with something akin to [`CheckCXXCompilerFlag`](https://cmake.org/cmake/help/latest/module/CheckCXXCompilerFlag.html) rather than depending on the version number. The Windows fork is just one example of where the version check is insufficient. Another would be the upstream clang compiler. The upstream releases of clang-13 and clang-14 are sufficient to build HIP language programs, but don't correspond to any particular ROCm version.

I know it's not the thing you're changing, but I also wanted to point out that this comparison can be simplified using tuples:
```
(hipccMaj, hipccMin) >= (5, 2)
```